--- C:\Users\Public\Development\ingrid\ingrid-iplug-csw-dsc\trunk\src\main\webapp\WEB-INF\spring.xml 
+++ (clipboard) 
@@ -326,7 +326,7 @@
     </bean>
     
     <bean id="xmlPersistency" class="de.ingrid.codelists.persistency.XmlCodeListPersistency">
-        <property name="pathToXml" value="data/codelistsNEW.xml" />    
+        <property name="pathToXml" value="data/codelists.xml" />    
     </bean>
     
     <bean id="codeListService" class="de.ingrid.codelists.CodeListService">
